github-actions/runtime-variables/RuntimeVariables.MD

```diff
@@ -0,0 +1,234 @@
+# Runtime Variables in Github Actions
```

Comment:
This content would probably be more appropriate to be under continous-deployment/recipes/github-actions-runtime-variables

Reply:
Done! I've changed the specific folder location after talking to Erik Schlegel.